rest/config.go

```diff
@@ -190,6 +190,7 @@ type DbConfig struct {
 	LocalDocExpirySecs        *uint32                        `json:"local_doc_expiry_secs,omitempty"`        // The _local doc expiry time in seconds
 	EnableXattrs              *bool                          `json:"enable_shared_bucket_access,omitempty"`  // Whether to use extended attributes to store _sync metadata
 	SessionCookieName         string                         `json:"session_cookie_name"`                    // Custom per-database session cookie name
+	SessionCookieHTTPOnly     bool                           `json:"http_only_cookies"`                      // HTTP only cookies
```

Comment:
Actually - one thing: Rename JSON tag for consistency (This is more important than the Go struct name, because it's exposed via the config API)

```suggestion
	SessionCookieHTTPOnly     bool                           `json:"session_cookie_http_only`                      // HTTP only cookies
```